### ISSUE TITLE
Prerequisite: Document most, if not all the methods in KekBot.

### ISSUE THREAD

**Issue by Godson777** [Godson777/KekBot#19]
This is to prevent any further confusion during development as to what something is supposed to do. I've found myself confused at times over "why did I do *this*, instead of *this*? Only to make the same mistake and have to undo it.

This also allows more ease of use to anyone who's willing to take advantage of the open source code.

**Comment by Godson777** [comment]
Already done. Had been forgotten about until now.
